org/netboot/tftp/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# https://github.com/jpillora/docker-dnsmasq/blob/master/Dockerfile
+
+FROM alpine:edge
+LABEL maintainer="dev@jpillora.com"
```

Comment:
For this Dockerfile too, it would be good to have an updated maintainer email for this container image